project/src/js/lichess/friends.js

```diff
@@ -1,8 +1,40 @@
-import { caseInsensitiveSort } from '../utils';
-
 var onlineFriends = [];
 
+var makeUser = function(name, isPlaying) {
+    return {
+      'name' : name,
+      'isPlaying' : isPlaying,
+    }
+};
+
+/** Compares usernames for equality, ignoring prefixed titles (such as GM) */
+var isSameUser = function(userId, name) {
```

Comment:
This should not be useful... the user string coming from `following_onlines` is already a `userId`.